README.md

```diff
@@ -101,15 +101,21 @@ Citing
 
 If you use pvlib-python in a published work, please cite:
 
-  William F. Holmgren, Clifford W. Hansen, and Mark A. Mikofski.
-  "pvlib python: a python package for modeling solar energy systems."
-  Journal of Open Source Software, 3(29), 884, (2018).
-  https://doi.org/10.21105/joss.00884
+  Anderson et al., (2023). pvlib python: 2023 project update.
```

Comment:
Could we list all the authors instead of using et al?  Maybe just last names, to save space.

Reply:
Certainly

Reply:
Has now been done. Let me know if you don't like the style of "Last name, First letter of first name"

Reply:
Can we do it here in README.md too?  Thanks :)